worker/utils/fastlyJob.js

```diff
@@ -12,6 +13,50 @@ class FastlyJobClass {
     }
   }
 
+  // takes in an array of urls and purges cache for each
+  async purgeCache(urlArray) {
+    if (!Array.isArray(urlArray)) {
+      throw new Error('Parameter `urlArray` needs to be an array of urls');
+    }
+
+    let urlCounter = urlArray.length;
+    let purgeMessages = [];
+
+    const headers = {
+      'fastly-key': environment.getFastlyToken(),
+      'accept': 'application/json',
+      'Fastly-Soft-Purge': '1',
```

Comment:
can you either create a var here with a name that suggests the purpose of this number, or put a comment here?

Reply:
will do